plugin/log_assert.py

```diff
@@ -177,7 +185,16 @@ def pytest_runtest_makereport(item, call):
 def get_logger(*args, **kwargs):
     if 'verbose_lvl' not in kwargs:
         kwargs['verbose_lvl'] = 0
-    return mrglog.get_logger(*args, **kwargs)
+    if 'output' not in kwargs:
+        kwargs['output'] = "std, txt"
```

Comment:
Shouldn't be ensured that when this output is used then StreamHandler must be among logger's handlers? (not sure, just asking)